src/main/java/org/sonarqube/A.java

```diff
@@ -3,7 +3,11 @@
 public class A {
   String message = "foo";
 
+  // TODO fix this
   public String foo() {
+    message = null;
+    message.length();
+
     return message;
```

Comment:
###AWAN8CfkIDTmpsjsOiqZ###
org.sonarqube:pr-decoration:src/main/java/org/sonarqube/A.java:BRANCH:pr_12@9
The return value of "length" must be used.

Reply:
@###@RESOLVED@###@

Reply:
@###@RESOLVED@###@

Reply:
@###@RESOLVED@###@